app/services/sso/jwt.rb

```diff
@@ -33,7 +33,7 @@ def jwt_token_from_request
     end
 
     def bearer_token_set?
-      request.authorization.present? && request.authorization.start_with?('Bearer')
+      request.authorization.present? && request.authorization.start_with?('JWT')
```

Comment:
This check no longer matches the function name. 
also, IIRC, jwt auth was added for graphql, would it still work with this change?

Reply:
Yes, sure. Graphql functionality should not have any impact from this implementation.

Reply:
Actually, this does break the functionality. But I'm not sure how we can avoid this.